IFC 4.3/SpanAnnotation_1/UT_SpanAnnotation_1.ifc

```diff
@@ -60,10 +60,15 @@ DATA;
 #40= IFCRELAGGREGATES('1HzNGEvrXFBAujvb88ukbA',#20,$,$,#35,(#39));
 #41= IFCDISTANCEEXPRESSION(35.0,$,$,$,.F.);
 #42= IFCLINEARSPANPLACEMENT($,#34,#41,$,$,285.0);
-#43= IFCPROPERTYSINGLEVALUE('CargoSpeed',$,IFCREAL(80.0),$);
-#44= IFCPROPERTYSINGLEVALUE('DesignSpeed',$,IFCREAL(160.0),$);
+#43= IFCPROPERTYSINGLEVALUE('CargoSpeed',$,IFCLINEARVELOCITYMEASURE(80.0),$);
+#44= IFCPROPERTYSINGLEVALUE('DesignSpeed',$,IFCLINEARVELOCITYMEASURE(160.0),$);
 #45= IFCPROPERTYSET('3rlxeJAgT4JOnfpRorgpF_',#20,'PSET_SpeedData',$,(#43,#44));
 #46= IFCRELDEFINESBYPROPERTIES('3QB20dNCj4Fu98jw9z5kIZ',#20,$,$,(#39),#45);
+#50= IFCDERIVEDUNIT((#51,#52),.LINEARVELOCITYUNIT.,$);
+#51= IFCDERIVEDUNITELEMENT(#53,1);
+#52= IFCDERIVEDUNITELEMENT(#54,-1);
+#53= IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#54= IFCSIUNIT(*,.TIMEUNIT.,$,.SECOND.);
```

Comment:
It seems a bit odd to define [m/s], but use 80 and 160 as speed values. Technically not wrong, but it may convey a wrong message to the rail experts reading this.

Reply:
```suggestion
#50= IFCDERIVEDUNIT((#51,#52),.LINEARVELOCITYUNIT.,$);
#51= IFCDERIVEDUNITELEMENT(#53,1);
#52= IFCDERIVEDUNITELEMENT(#54,-1);
#53= IFCSIUNIT(*,.LENGTHUNIT.,.KILO.,.METRE.);
#54= IFCCONVERSIONBASEDUNIT(#55,.TIMEUNIT.,'hour',#56);
#55=IFCDIMENSIONALEXPONENTS(0, 1, 0, 0, 0, 0, 0);
#56=IFCMEASUREWITHUNIT(IFCTIMEMEASURE(3600), #26);
```

Hopefully, I didn't screw up the syntax.

Reply:
please review the recent commit c6284ef integrating the requested units

Reply:
LFTM.